cmd/cli/cmd/util.go

```diff
@@ -246,21 +246,22 @@ type ActionResponse struct {
 }
 
 func (a *ActionResponse) Print() {
+	ok := a.Code == "0" || a.Code == ""
 	if WantJsonOutput {
-		if a.Code == "0" || a.Code == "" {
+		if ok {
 			a.Code = "ok"
 		}
 		dump, err := json.Marshal(a)
 		if err != nil {
 			log.Fatal(err)
 		}
 		log.SetFlags(log.Flags() &^ (log.Ldate | log.Ltime))
-		log.Fatal(string(dump))
+		log.Print(string(dump))
```

Comment:
This is OBE by AC-611 but I'll approve and update AC-611 PR.

Reply:
:+1: